hack/kubestellar.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 (start or stop | -v)
+
+# Purpose: deploy the kubestellar platform. The following components are created:
```

Comment:
"deploy" is a vague word, it could cover installing the executables --- which this command does NOT do.  I would reword this something along the following lines.

> # Purpose: ensure that the ESPW is set up and the controllers are running.

Reply:
Fixed